test/support/utils.rb

```diff
@@ -91,6 +89,22 @@ def create_pseudo_terminal(boot_options: "-r debug/run")
 
     private
 
+    # use this to start a debug session with the test program
+    def start_debugger_with(program)
+      write_temp_file(strip_line_num(program))
+      inject_lib_to_load_path
+      pid = spawn("#{RUBY} -r debug/open #{temp_file_path}")
```

Comment:
It is kind to show the prompt (ex) `"[PLEASE DEBUG with DEBUGGER @#{caller.fisst}]"`)

Reply:
added (you can see it from the description's screenshot)